### DISCUSSION TITLE
Add-ons __INIT__

### DISCUSSION THREAD

**Discussion by mac4kent** [nutti/Screencast-Keys#207] [Q&A]
I would like to know about __INIT__ script because my friend ask me take test. I don't know how to use .py and __init__ code.

**Comment by nutti** [comment]
This is not related to the Screencast Keys.
You can ask the Python general question at anywhere like stackoverflow or Reddit.

**Reply by mac4kent** [reply]
I did not mean screencast key. I talk about __init__. I don't know where to find question/answer on Reddit and stackoverflow.

**Reply by nutti** [reply]
Like this

https://stackoverflow.com/questions/8609153/why-do-we-use-init-in-python-classes

Anyway, this discussion should not be done here.
You can get more precise answer at stackoverflow or Reddit.
If you don't find the answer, you can also ask at there.

**Reply by mac4kent** [reply]
ok many thanks.
